_sass/components/_btn.scss

```diff
@@ -45,7 +61,7 @@
 }
 
 @include at-media-max("desktop") {
-  .btn-lang-toggle.mobile {
+  div.btn-lang-toggle {
```

Comment:
when do we use this toggle on something that isn't a `<div>`

Reply:
> when do we use this toggle on something that isn't a `<div>`

`btn-lang-toggle` is an existing (albeit not-so-accurate) class name used for both the static label + list (mobile) and button + dropdown (desktop) language picker variations:

https://github.com/18F/identity-site/blob/83943a8a6a8747c1f15ec1809c48a45c3a6128db/_includes/header.html#L45-L50

![image](https://user-images.githubusercontent.com/1779930/103814642-0788f300-5030-11eb-8e3b-979392204b4f.png)

https://github.com/18F/identity-site/blob/83943a8a6a8747c1f15ec1809c48a45c3a6128db/_includes/language_search.html#L16-L21

![image](https://user-images.githubusercontent.com/1779930/103814653-0d7ed400-5030-11eb-823c-c317c76c099d.png)

Because I dropped the `.mobile` class here, `div` was the next best thing to assign styles for the existing element.

Separately, I'd think we could probably rename this to something which describes both cases equally well if we wanted to share styles. Something like `language-picker__label`.

Reply:
ok! I think the `.mobile` class makes the intent clearer? But this is not my wheelhouse

Reply:
> ok! I think the `.mobile` class makes the intent clearer? But this is not my wheelhouse

Yeah, I'd intended to try to refactor this a bit to treat it more as "list form" vs. "dropdown form" (as opposed to "desktop" vs. "mobile"), but in any case, a class would likely be clearer. I'll plan to make an update.